{{cookiecutter.plugin_name}}/src/{{cookiecutter.module_name}}/_tests/test_dock_widget.py

```diff
@@ -1,18 +1,41 @@
-import {{cookiecutter.module_name}}
+from {{cookiecutter.module_name}} import ExampleQWidget, example_magic_widget
+import numpy as np
 import pytest
 
-# this is your plugin name declared in your napari.plugins entry point
-MY_PLUGIN_NAME = "{{cookiecutter.plugin_name}}"
-# the name of your widget(s)
-MY_WIDGET_NAMES = ["Example Q Widget", "example_magic_widget"]
+# we create a fixture that returns us a random image array
```

Comment:
i'm wondering if you're adding this in lieu of our recent "let's teach them fixtures!" conversation? :)

this feels a bit gratuitous to me :) over `viewer.add_image(np.random.random((100, 100)))`

Reply:
I actually added it because I'd called `np.random.random(100, 100)` rather than `np.random.random((100, 100))` and then had to change it in two places - *then* I thought about our recent conversation 🤣.

But yes you're probably right, not really a meaningful example.